File: public/dsa/searching/search_rotated_sorted_array/naive/search_rotated_sorted_array.py
from typing import List


class Solution:
    def getElementIndex(self, array: List[int], target: int) -> int:
        for i in range(len(array)):
            if array[i] == target:
                return i
        return -1


# tests-start
array = [4, 5, 6, 7, 1, 2, 3]
target = 6
sol = Solution()
print(sol.getElementIndex(array, target))
# tests-end

'''output
2
'''

File: public/dsa/searching/search_rotated_sorted_array/naive/search_rotated_sorted_array.cpp
#include <bits/stdc++.h>
using namespace std;

int getElementIndex(vector<int> &arr, int target) {
    for (int i = 0; i < arr.size(); i++) {
        if (arr[i] == target) {
            return i;
        }
    }
    return -1;
}

int main() {
    vector<int> arr = {4, 5, 6, 7, 1, 2, 3};
    int target = 6;
    int index = getElementIndex(arr, target);
    cout << index << endl;
}

/*output
2
*/